slack-java-client/src/main/java/com/hubspot/slack/client/SlackClientModule.java

```diff
@@ -7,6 +7,8 @@
 public class SlackClientModule extends AbstractModule {
   @Override
   protected void configure() {
+    binder().requireAtInjectOnConstructors();
```

Comment:
this caused issues with any class that has 0 arg constructors. can we drop this line?